README.md

```diff
@@ -21,6 +21,40 @@ This project exposes a REST API with the following endpoints:
 
 More information available in [api.md](blob/master/api.md).
 
+## Deployment
+In order to deploy the site, ensure you have Ansible 2.2x installed. You will
+be prompted for a "Vault password" when running most of these commands. Look
+in the "Image Annotator" note in LastPass for the password.
```

Comment:
hows this?

In order to deploy the site, ensure you have Ansible 2.2x installed. You will
be prompted for a "Vault password" when running most of these commands. Any of
committer on this project should be able to help!

If you work at Bocoup, you can find the password in the "Image Annotator" note
on our LastPass account.